### ISSUE TITLE
AND filter isn't working

### ISSUE THREAD

**Issue by savannahostrowski** [Azure/awesome-azd#136]
<img width="1217" alt="image" src="https://github.com/Azure/awesome-azd/assets/8949415/8ef0a757-4111-41bf-9f5c-2d8ccaa286b4">

**Comment by gkulin** [comment]
@hemarina could you take a look at this?

Savannah said she can't reliably repro it but I was able to:
<img width="1603" alt="image" src="https://github.com/Azure/awesome-azd/assets/55554236/80313c14-147a-4ecb-a854-aec8f9fc46d8">

**Comment by rajeshkamal5050** [comment]
Yeah, looks like when you keep toggling between OR and AND. The card panel is getting duplicated and added.
